scripts/bad-linebreaks.mjs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+import fs from 'fs';
+import { glob } from 'glob';
+
+function getLine(txt, index) {
+
+  let line = 1;
+
+  for (let i = 0; i < index; i++) {
+    if (txt[i] === '\n') {
+      line += 1;
+    }
+  }
+
+  return line;
+
+}
+
+export function findBadLinebreaks(file, fix = false) {
+
+  let contents = fs.readFileSync(file, 'utf8').toString();
+
+  const re = /(?<=[\w\d ])\n(?=[\w\d])/g;
+  const matches = Array.from(contents.matchAll(re));
+
+  for (const m of matches) {
+
+    if (fix) {
+      contents = `${contents.substring(0, m.index)} ${contents.substring(m.index + 1)}`;
```

Comment:
```suggestion
      contents = `${contents.slice(0, m.index)} ${contents.slice(m.index + 1)}`;
```
lol, this would actually be nice to have `contents.with(m.index, ' ')` for

Reply:
In case of `abc \nefg` this will create `abc  efg`.
```suggestion
      contents = `${contents.slice(0, m.index)}.trimEnd() ${contents.slice(m.index + 1)}`;
```

Reply:
I was thinking this would get run after mdlint fixes, which would trim the ends, but sure, why not